apps/usgsearthquakes/usgs_earthquakes.star

```diff
@@ -239,10 +255,11 @@ def fetch_earthquakes(lat, lng, radius, magnitude):
         return None
 
     features = resp.json().get("features")
-    if not features:
-        # buildifier: disable=print
-        print("missing features: %s" & resp.body())
-        return None
+
+    #2024-09-27# if not features:
+    # buildifier: disable=print
+    #2024-09-27#    print("missing features: %s" + resp.body())
+    #2024-09-27#    return None
```

Comment:
suggest just deleting these lines if we don't need them.

Reply:
Yep